services/pypi/pypi-helpers.js

```diff
@@ -50,6 +50,27 @@ const parseClassifiers = function(parsedData, pattern) {
   return results
 }
 
+function getLicenses(packageData) {
+  const {
+    info: { license },
+  } = packageData
+  if (license) {
+    return [license]
+  } else {
+    const acronymRegex = /\([^)+]\)/
```

Comment:
Don't we want the `+` to be after the `[`?

Reply:
Ha, yea, that is totally broken. We don't want the parentheses, either.